components/00-mixins/opening-hours/_opening-hours.scss

```diff
@@ -7,16 +7,16 @@
 
   > .openinghours {
     padding-right: 0;
-    padding-bottom: $month-day-size-mobile / 2;
+    padding-bottom: $month-day-size-mobile *.5;
     border: 0;
   }
 
   .openinghours--details {
     @include theme('border-color', 'color-primary--lighten-3', 'opening-hours-border');
 
-    top: calc(100% + #{$month-day-size-mobile / 2});
-    left: $month-day-size-mobile / 4;
-    padding-top: $month-day-size-mobile / 2;
+    top: calc(100% + #{$month-day-size-mobile *.5});
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.

```sass
 top: calc(100% + #{$month-day-size-mobile * .5});
```